tools/cldr-code/src/main/java/org/unicode/cldr/util/personname/PersonNameFormatter.java

```diff
@@ -553,22 +618,40 @@ public static ParameterMatcher from(String string) {
         @Override
         public String toString() {
             List<String> items = new ArrayList<>();
-            if (!lengths.isEmpty()) {
-                items.add("length=" + lengths);
-            }
-            if (!styles.isEmpty()) {
-                items.add("style=" + styles);
-            }
-            if (!usages.isEmpty()) {
-                items.add("usage=" + usages);
+            showAttributes("length", lengths, items);
+            showAttributes("style", styles, items);
+            showAttributes("usage", usages, items);
+            showAttributes("order", orders, items);
+            return items.isEmpty() ? "ANY" : "{" + JOIN_SPACE.join(items) + "}";
+        }
+
+        public String toLabel() {
+            StringBuilder sb  = new StringBuilder();
+            addToLabel(lengths, sb);
+            addToLabel(lengths, sb);
```

Comment:
shouldn't this be styles, not lengths? And the next ones usages and orders?

Reply:
good catch fixed this. Also fixed the identifiers

Reply:
You can just look at the commits after this one for changes:
https://github.com/unicode-org/cldr/pull/1798/commits/f64d51655f40a7324819a72d145d345fe5410849
https://github.com/unicode-org/cldr/pull/1798/commits/11fae69c6d9e8d6c3157dd4e8388f9a0f4e5e6d1